src/Core/Domain/Security/Authentication/AuthenticationException.php

```diff
@@ -31,4 +31,12 @@ public static function notAuthenticated(): self
     {
         return new self(_('Authentication failed'));
     }
+
+    /**
+     * @return self
+     */
+    public static function userBlocked(): self
+    {
+        return new self(_('User is blocked'));
+    }
```

Comment:
Could it be possible to add the blocking duration to the message ?

Reply:
this cannot well formatted and cannot manage locale